lib/options.go

```diff
@@ -178,10 +178,20 @@ func (ipnet *IPNet) UnmarshalText(b []byte) error {
 	}
 
 	*ipnet = *newIPNet
-
 	return nil
 }
 
+// MarshalJSON returns the JSON representation of IPNet using CIDR notation.
+func (ipnet *IPNet) MarshalJSON() ([]byte, error) {
+	return []byte(fmt.Sprintf("%q", ipnet.String())), nil
+}
+
+// UnmarshalJSON decodes a JSON representation of IPNet using CIDR notation.
+func (ipnet *IPNet) UnmarshalJSON(b []byte) error {
+	// Remove quotes and fallback to unmarshal text
+	return ipnet.UnmarshalText([]byte(strings.Trim(string(b), `"`)))
+}
+
```

Comment:
I think this is fine, but in reality, all you need is MarshalText as it seems that we fall back to that and if you just replace all of that with MarshalText that just return `[]byte(ipnet.String())` it seems still to work.